Classes/Configuration/BackendConfigurationManager.php

```diff
@@ -14,6 +12,11 @@ class BackendConfigurationManager extends \TYPO3\CMS\Extbase\Configuration\Backe
     protected function getCurrentPageIdFromGetPostData(): int
     {
         // XHR requests have page ID in "page"
-        return (int)GeneralUtility::_GP('page') ?? parent::getCurrentPageIdFromGetPostData();
+        return (int)\TYPO3\CMS\Core\Utility\GeneralUtility::_GP('page') ?? parent::getCurrentPageIdFromGetPostData();
```

Comment:
Wouldn't this be better with an import? `use TYPO3\CMS\Core\Utility\GeneralUtility;`

```suggestion
        return (int)GeneralUtility::_GP('page') ?? parent::getCurrentPageIdFromGetPostData();
```